sound/soc/sof/intel/cnl.c

```diff
@@ -65,12 +65,6 @@ static irqreturn_t cnl_ipc_irq_thread(int irq, void *context)
 			snd_sof_ipc_reply(sdev, msg);
 		}
 
```

Comment:
please fix commit messages using the standard pattern of describing the issue, how you fix it and why it's safe to do so, and don't forget about prefixes e.g.

ASoC: SOF: Intel: hda: remove unnecessary waitq when loading firmware

The use of a waitq while loading code is unnecessary. The ROM IPC which provides the 'PURGE_FW' information is already handled with a timeout in cl_dsp_init. As a result the additional wait before downloading firmware is redundant and can be removed safely.

Reply:
@plbossart  thank you for your review and suggestions, Pierre, the commit messages are fixed now.